docs/sources/dashboards/variables/add-template-variables/index.md

```diff
@@ -87,32 +87,32 @@ Query variables are generally only supported for strings. If your query returns
 Query expressions can contain references to other variables and in effect create linked variables. Grafana detects this and automatically refreshes a variable when one of its linked variables change.
 
 {{% admonition type="note" %}}
-Query expressions are different for each data source. For more information, refer to the documentation for your [data source]({{< relref "../../../datasources/" >}}).
+Query expressions are different for each data source. For more information, refer to the documentation for your [data source][].
```

Comment:
Different link, similar question, how important is the link to [Datasources](https://grafana.com/docs/grafana/latest/datasources/)?